ota-plus-web/app/com/advancedtelematic/api/clients/UserProfileApi.scala

```diff
@@ -110,11 +110,17 @@ class UserProfileApi(val conf: Configuration, val apiExec: ApiClientExec)(implic
       .withUser(userId)
       .execJsonValue(apiExec)
 
-  def organizationRequest(namespace: Namespace, userId: UserId, method: String, path: String, body: Option[JsValue])
-                         (implicit traceData: TraceData): Future[Result] =
+  def organizationRequest(namespace: Namespace,
+                          userId: UserId,
+                          method: String,
+                          path: String,
+                          queryParams: Map[String, String],
+                          body: Option[JsValue]
+                         )(implicit traceData: TraceData): Future[Result] =
     userProfileRequest(s"organizations/${segment(namespace.get)}/$path")
       .transform(_.withMethod(method))
-      .transform(r => body.map(json => r.withBody(json)).getOrElse(r))
+      .transform(_.withQueryStringParameters(queryParams.toSeq: _*))
```

Comment:
I had to add the query parameters here because the request to evict a user from an org sends the email of the user in a query parameter.